tools-public/toolpad.yaml

```diff
@@ -112,5 +119,869 @@ nodes:
+      dataSource:
+        type: const
+        value: rest
+    name: openPRs
+    id: dlm3hqd
+    type: query
+    parentId: 39j3h6b
+    parentProp: queries
+    parentIndex: a0
+    params:
+      - - repository
+        - type: jsExpression
+          value: |
+            page.parameters.repository
+  fyq3hur:
+    name: PageRow
+    props: {}
+    attributes:
+      component:
+        type: const
+        value: PageRow
+    layout: {}
+    id: fyq3hur
+    type: element
+    parentId: 39j3h6b
+    parentProp: children
+    parentIndex: a0
+  xup3hw6:
+    name: codeComponent_HealthBadge
+    props:
+      value:
+        type: jsExpression
+        value: |
+          openPRs.data.total_count
+      unit:
+        type: const
+        value: " open PRs"
+      warning:
+        type: const
+        value: 50
+      problem:
+        type: const
+        value: 75
+      lowerIsBetter:
+        type: const
+        value: true
+    attributes:
+      component:
+        type: const
+        value: codeComponent.HealthBadge
+    layout: {}
+    id: xup3hw6
+    type: element
+    parentId: fyq3hur
+    parentProp: children
+    parentIndex: a0
+  9ou3hb2:
+    name: medianTimeToCompletion
+    attributes:
+      title:
+        type: const
+        value: page
+      parameters:
+        type: const
+        value:
+          - - repoId
+            - "23083156"
+    id: 9ou3hb2
+    type: page
+    parentId: 9q8eshxx
+    parentProp: pages
+    parentIndex: a6
+  4jx3hv9:
+    attributes:
+      query:
+        type: const
+        value:
+          method: GET
+          headers: []
+          browser: false
+          url:
+            type: jsExpression
+            value: >
+              `https://api.ossinsight.io/q/analyze-pull-request-open-to-merged?repoId=${parameters.repoId}`
+      connectionId:
+        type: const
+        value: null
+      dataSource:
+        type: const
+        value: rest
+    name: pullRequestOpenToMerged
+    id: 4jx3hv9
+    type: query
+    parentId: 9ou3hb2
+    parentProp: queries
+    parentIndex: a0
+    params:
+      - - repoId
+        - type: jsExpression
+          value: |
+            page.parameters.repoId
+  e8113hoz:
+    name: PageRow
+    props: {}
+    attributes:
+      component:
+        type: const
+        value: PageRow
+    layout: {}
+    id: e8113hoz
+    type: element
+    parentId: 9ou3hb2
+    parentProp: children
+    parentIndex: a0
+  xv103hin:
+    name: codeComponent_HealthBadge
+    props:
+      value:
+        type: jsExpression
+        value: >
+          (() => {
+            const length = pullRequestOpenToMerged.data.data.length
+
+            return (
+              Math.round((pullRequestOpenToMerged.data.data[length - 1].p75 / 24) * 100) /
+              100
+            )
+          })()
+      warning:
+        type: const
+        value: 3
+      problem:
+        type: const
+        value: 0
+      unit:
+        type: const
+        value: days
```

Comment:
I think that there should be a space here

```suggestion
        value: ' days'
```

<img width="402" alt="Screenshot 2023-03-22 at 23 32 50" src="https://user-images.githubusercontent.com/3165635/227053336-5197d051-2ef7-4ba5-ad34-0bd0e7bd4e47.png">

Reply:
Or maybe the other way, around, fix the badge component.